pages/linux/cmus.md

```diff
@@ -24,7 +24,7 @@
 
 `s`
 
-- Turn volume up/down and seeking
+- Turn volume up/down and seeking:
```

Comment:
I can't understand the meaning behind this, and it also seems `seeking` doesn't agree gramatically with the rest of the sentence. What's the context of `seeking` here?

Reply:
The up/down arrow keys are for volume. And the left/right keys are for seeking.

I would separate the two into separate examples.